xlive/H2MOD/GUI/imgui_integration/Console/CommandsUtil.h

```diff
@@ -193,25 +193,31 @@ class CircularStringBuffer
 		va_list valist;
 		va_start(valist, fmt);
 		int buffer_size_needed = _vsnprintf(NULL, 0, fmt, valist) + 1;
-		if (buffer_size_needed < m_line_buf_size)
+		if (buffer_size_needed != -1)
```

Comment:
Will be -1 only when the result is -2. Practically never, because that function only returns - 1 if the buffer is insufficient.

Reply:
changed so it makes sure the value is positive, need this since we cast it to a size_t type afterwards and don't want it to cause undefined behaviour.